### ISSUE TITLE
Fine-tuning Code

### ISSUE THREAD

**Issue by siny1998** [ljwztc/CLIP-Driven-Universal-Model#80]
Hi,

Excellent work on the model!
I am interested in fine-tuning your pre-trained model (SwinUNETR) using my own dataset. Could you please provide more details regarding the hyperparameters for downstream tasks? Specifically, I would like to know about the recommended learning rate and optimizer.

Thank you in advance for your assistance.

**Comment by MrGiovanni** [comment]
Hi @siny1998 

Here is an example of our fine-tuning code on TotalSegmentator.
https://github.com/MrGiovanni/SuPreM/tree/main/target_applications/totalsegmentator

Best,
Zongwei

**Comment by siny1998** [comment]
Dear Zhou,

Thank you very much for the link you provided. It has been immensely helpful.

Best regards,
Siny
